npm/webpack-preprocessor/package.json

```diff
@@ -20,6 +20,7 @@
     "watch": "rimraf dist && tsc --watch"
   },
   "dependencies": {
+    "@babel/parser": "7.13.0",
```

Comment:
why does this need @babel/parser?

Reply:
https://github.com/cypress-io/cypress/blob/ryanm/fix/improve-binary-cleanup-to-look-at-binary-rather-than-monorepo/npm/webpack-preprocessor/lib/cross-origin-callback-loader.ts#L2